provider/labels.go

```diff
@@ -3,8 +3,10 @@
 package gcp
 
 import (
+	"context"
```

Comment:
Std lib should be a separate import block.
```suggestion
	"context"
	
```